sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/FileSystem.java

```diff
@@ -136,7 +136,9 @@
   /**
    * Files matching the given predicate.
    * @see #predicates()
+   * @deprecated since 6.6 Plugins should avoid working with {@link File} and prefer working with {@link InputFile}
    */
+  @Deprecated
   Iterable<File> files(FilePredicate predicate);
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)